unit-tests/live/options/test-timestamp-domain.py

```diff
@@ -0,0 +1,96 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+# test:device D400*
+
+import time
+import pyrealsense2 as rs
+
+from rspy import test
+
+
+def close_resources(sensor):
+    """
+    Stop and Close sensor.
+    :sensor: sensor of device
+    """
+    if len(sensor.get_active_streams()) > 0:
+        sensor.stop()
+        sensor.close()
+
+
+def set_and_verify_timestamp_domain(sensor, global_time_enabled: bool):
+    """
+    Perform sensor (depth or color) test according given global time
+    :sensor: depth or color sensor in device
+    :global_time_enabled bool: True - timestamp is enabled otherwise false
+    """
+    global frame_queue
+
+    try:
+        sensor.set_option(rs.option.global_time_enabled, 1 if global_time_enabled else 0)
+        time.sleep(0.7)
+        frame = frame_queue.wait_for_frame()
+
+        if frame_queue is None or frame_queue.size() == 0:
```

Comment:
if Not frame: ?

Reply:
"if" statement changed to:
```python
    if not frame or frame_queue is None or frame_queue.size() == 0:
```

Reply:
Lets keep it simple, at this point of the code we only care about the frame right?
`if not frame:`

Reply:
`wait_for_frame()` function has timeout 5 sec. It raise Exception if not received frame?

Reply:
Yes, you are correct. so we don't even need `if not frame:` :)

Reply:
If we will must to do one addition commit I will remove `if not frame:` ...